packages/services/src/balances/sip10-balances.service.ts

```diff
@@ -55,6 +57,7 @@ export function createSip10BalancesService(
     return {
       usd: totalUsdBalance,
       addressBalances,
+      aggregateBalances: getAggregateSip10Balances(addressBalances),
```

Comment:
@alexp3y I added `aggregateBalances` as that's what we need on the homepage. 

We don't use `addressBalances` from this service but it could be useful? I'm not sure. I think it has a different query cache than account specific balances anyway?